proc_nutation.py

```diff
@@ -33,13 +34,14 @@
         searchstr,
         exp_type=exp_type,
         expno=nodename,
-        postproc=postproc,
+        postproc=postproc,# PR why do you need to manually specify the postproc? that should not be needed!
```

Comment:
deal with this

Reply:
again wrong script outside of examples not sure what this is so deleted. Correct version inside examples repo